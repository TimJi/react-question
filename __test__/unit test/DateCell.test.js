import { render } from '@testing-library/react'
import DateCell from '../../components/DateRanger/DataCell'

describe('DateCell', () => {
  it('should render correctly', () => {
    const { container } = render(<DateCell date={'3'} />)
    expect(container)
      .toMatchSnapshot('')
  })
})
