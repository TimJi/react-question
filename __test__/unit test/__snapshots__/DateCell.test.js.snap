// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DateCell should render correctly 1`] = `
<div>
  <div
    class="w-[50px] h-9 bg-white hover:bg-[#e6e6e6]"
  >
    3æ—¥
  </div>
</div>
`;
